Foody/Onboarding/Login/LoginViewController.swift

```diff
@@ -6,29 +6,99 @@
 //
 
 import UIKit
+import FirebaseAuth
+import Firebase
 
 class LoginViewController: UIViewController {
 
+    @IBOutlet weak var loginImageView: UIImageView!
+    @IBOutlet weak var loginLbl: UILabel!
+    @IBOutlet weak var welcomeLbl: UILabel!
+    @IBOutlet weak var rememberMeSwitch: UISwitch!
+    @IBOutlet weak var emailTextFld: UITextField!
+    @IBOutlet weak var passwordTextFld: UITextField!
+    @IBOutlet weak var loginBttn: UIButton!
+    @IBOutlet weak var registerBttn: UIButton!
+    
     var clickedRegisterCompletion:(() -> ())?
+    let defaults: UserDefaults? = UserDefaults.standard
```

Comment:
Why it is optional? No need to be optional.  You can change 
```suggestion
    let defaults = UserDefaults.standard
```